### PR TITLE
refactor: Renamed USE_SAML2 to use_saml2

### LINKED ISSUE #2132: keys in json shoud be lowercase written
We should rename USE_SAML2 to use_saml2 in lowercase, or have a better name in lowercase

If there are others in Upper of our codebase they also need to become renamed.

### PR DESCRIPTION
**Pull request by Beeram12**
renamed USE_SAML2 to use_saml2 lowered all .

Fixes #2132

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
Please rename only the USE_SAML2 in the json output. Not any constant we have.

File: mslib/mscolab/server.py [hunk 1]
```diff
@@ -242,16 +242,16 @@ def hello():
             auth.login_required()
             return json.dumps({
                 'message': "Mscolab server",
-                'USE_SAML2': mscolab_settings.USE_SAML2
+                'use_saml2': mscolab_settings.use_saml2
```
Comment (ReimarBauer) [inline mslib/mscolab/server.py:245]: this is json :)

File: mslib/mscolab/server.py [hunk 2]
```diff
@@ -732,7 +732,7 @@ def reset_request():
         return render_template('errors/403.html'), 403
 
 
-if mscolab_settings.USE_SAML2:
+if mscolab_settings.use_saml2:
```
Comment (ReimarBauer) [inline mslib/mscolab/server.py:735]: this is a constant, this should not be renamed

File: mslib/mscolab/mscolab.py [hunk 3]
```diff
@@ -477,7 +477,7 @@ def main():
                     This will generate necessary metada data file for sso in mscolab through localhost idp
 
                     Before running this function:
-                    - Ensure that USE_SAML2 is set to True.
+                    - Ensure that use_saml2 is set to True.
```
Comment (ReimarBauer) [inline mslib/mscolab/mscolab.py:480]: this is not json

File: mslib/mscolab/conf.py [hunk 4]
```diff
@@ -115,7 +115,7 @@ class default_mscolab_settings:
     GDPR = None
 
     # enable login by identity provider
-    USE_SAML2 = False
+    use_saml2 = False
```
Comment (ReimarBauer) [inline mslib/mscolab/conf.py:118]: this is not json

Comment (ReimarBauer) [reply mslib/mscolab/conf.py:118]: Do you still work on this @Beeram12  ?

Comment (Beeram12) [reply mslib/mscolab/conf.py:118]: @ReimarBauer Sorry for the delay,I am unaware of json format so I couldn't give update regarding this but will come up with right pull request.Kindly forgive me for many mistakes I am a beginner to open source.

Comment (ReimarBauer) [reply mslib/mscolab/conf.py:118]: Only where we use the json module you need to change it. Not on all places. 

https://www.w3schools.com/python/python_json.asp

**Pull request closed without merging**